src/controllers/userController.js

```diff
@@ -64,5 +64,19 @@ const User = {
       return sendResult(res, 500, `it is not you, it is us\n${error.message}`);
     }
   },
+  async OauthLogin(req, res) {
+    const [data] = await db.Users.findOrCreate({
+      where: { email: req.user.email },
+      defaults: req.user
+    });
+
+    return sendResult(res, 201, 'User logged successfully', {
+      userId: data.id,
+      username: data.username,
+      email: data.email,
+      token: helpers.createToken(data.id, data.email, data.isverified)
```

Comment:
Destructure `id`, `email`,`isverified` and `username` from data

Reply:
I corrected it